lib/filewatch/tail.rb

```diff
@@ -42,7 +42,8 @@ def initialize(opts={})
         :stat_interval => 1,
         :discover_interval => 5,
         :exclude => [],
-        :start_new_files_at => :end
+        :start_new_files_at => :end,
+        :delimiter => "\n"
```

Comment:
Just asking, as it might make sense (most probably) to use an independant system delimiter ... not sure if it makes sense in the context of this gem, anyway, jejeje.